src/pudl/package_data/phmsagas/column_maps/yearly_miles_of_pipe_in_state_by_material.csv

```diff
@@ -0,0 +1,102 @@
+total_total_steel_cathodically_protected_bare,,,,,,,,,,,,,,,,,,,,,partpcpbtotal,partpcpbtotal,partpcpbtotal,partpcpbtotal,partpcpbtotal,partpcpbtotal,partpcpbtotal,partpcpbtotal,partpcpbtotal,partpcpbtotal,partpcpbtotal,partpcpbtotal,partpcpbtotal
+total_total_steel_cathodically_protected_coated,,,,,,,,,,,,,,,,,,,,,partpcpctotal,partpcpctotal,partpcpctotal,partpcpctotal,partpcpctotal,partpcpctotal,partpcpctotal,partpcpctotal,partpcpctotal,partpcpctotal,partpcpctotal,partpcpctotal,partpcpctotal
+total_total_steel_cathodically_unprotected_bare,,,,,,,,,,,,,,,,,,,,,partpcubtotal,partpcubtotal,partpcubtotal,partpcubtotal,partpcubtotal,partpcubtotal,partpcubtotal,partpcubtotal,partpcubtotal,partpcubtotal,partpcubtotal,partpcubtotal,partpcubtotal
+total_total_steel_cathodically_unprotected_coated,,,,,,,,,,,,,,,,,,,,,partpcuctotal,partpcuctotal,partpcuctotal,partpcuctotal,partpcuctotal,partpcuctotal,partpcuctotal,partpcuctotal,partpcuctotal,partpcuctotal,partpcuctotal,partpcuctotal,partpcuctotal
+total_total_cast_iron,,,,,,,,,,,,,,,,,,,,,partpcitotal,partpcitotal,partpcitotal,partpcitotal,partpcitotal,partpcitotal,partpcitotal,partpcitotal,partpcitotal,partpcitotal,partpcitotal,partpcitotal,partpcitotal
+total_total_wrought_iron,,,,,,,,,,,,,,,,,,,,,partpwitotal,partpwitotal,partpwitotal,partpwitotal,partpwitotal,partpwitotal,partpwitotal,partpwitotal,partpwitotal,partpwitotal,partpwitotal,partpwitotal,partpwitotal
+total_total_plastic,,,,,,,,,,,,,,,,,,,,,partpptotal,partpptotal,partpptotal,partpptotal,partpptotal,partpptotal,partpptotal,partpptotal,partpptotal,partpptotal,partpptotal,partpptotal,partpptotal
+total_total_composite,,,,,,,,,,,,,,,,,,,,,partpctotal,partpctotal,partpctotal,partpctotal,partpctotal,partpctotal,partpctotal,partpctotal,partpctotal,partpctotal,partpctotal,partpctotal,partpctotal
+total_total_other,,,,,,,,,,,,,,,,,,,,,partpototal,partpototal,partpototal,partpototal,partpototal,partpototal,partpototal,partpototal,partpototal,partpototal,partpototal,partpototal,partpototal
+total_total_total,,,,,,,,,,,,,,,,,,,,,partptotalmiles,partptotalmiles,partptotalmiles,partptotalmiles,partptotalmiles,partptotalmiles,partptotalmiles,partptotalmiles,partptotalmiles,partptotalmiles,partptotalmiles,partptotalmiles,partptotalmiles
+other_materials,,,,,,,,,,,,,,,,,,,,,partpothermaterials,partpothermaterials,partpothermaterials,partpothermaterials,partpothermaterials,partpothermaterials,partpothermaterials,partpothermaterials,partpothermaterials,partpothermaterials,partpothermaterials,partpothermaterials,partpothermaterials
+gathering_other_materials,,,,,,,,,,,,,,,,,,,,,partpgothermaterials,partpgothermaterials,partpgothermaterials,partpgothermaterials,partpgothermaterials,partpgothermaterials,partpgothermaterials,partpgothermaterials,partpgothermaterials,partpgothermaterials,partpgothermaterials,partpgothermaterials,partpgothermaterials
```

Comment:
So for most of the field names, we have `part<letter><t or g or nothing><name>` - so I inferred that these were the "total other materials" and "gathering other materials" columns... but I think it sort of makes sense for these to be the "transmission other materials" & "gathering other materials" columns - to show what those other materials might be. I guess `partpgothermaterials` is pretty unambiguously "gathering", so my question is - do we think we should call this `total_other_materials` or `transmission_other_materials`?

Reply:
Ugh that's tough. I think I would go with `total_other_materials` because it is not specified.... even though with more context we might learn that this is transmission not total.

It looks like these two are outside of the core table in the PDF version of the table. The description is "Composite pipe requires a PHMSA Special Permit or waiver from a State2 specify Other material(s):"
I would definitely add something about `special_permit` or something to the names of both of these.
So I

Reply:
I read the PDF as saying two things:

1. Composite pipe requires a special permit / waiver
2. please specify "other materials" - we'll store this as `partpothermaterials` and `partpgothermaterials`

In which case we shouldn't have any `composite`/`special_permit` in the name.

Part of why I read it that way is that the rest of the form distinguishes between `composite` & `other` materials, and another is that there's a little superscript (1) by the Composite column header which seems like it would correspond to the first point.

<img width="121" alt="image" src="https://github.com/catalyst-cooperative/pudl/assets/2495794/7c0f2b94-bb7d-4ece-bde2-d167aec77942">

<img width="354" alt="image" src="https://github.com/catalyst-cooperative/pudl/assets/2495794/b8f55c36-7b6d-4a44-835e-477c3e952cc5">

What do you think?

Reply:
Just a heads up that when there's a "specify" field I've been calling it "x_y_description" as a column name. (E.g., "gathering_other_materials_description").

Reply:
oh yes you are totally right here. good eye. no `composite` or `special_permit`!

hm okay well that changes my answer to you were originally asking because now we can check if `partpototal` shows up when there is any other pipe or just transmission and it looks like there are only values in this `partpototal` when there is other transmission miles! so i would go with `transmission_other_materials`

Reply:
Sweet, thanks for all the discussion! Going with `transmission/gathering_other_materials_description` :)